### ISSUE TITLE
`Install-SQLMonitor` - Step `2__AllDatabaseObjects` - Error 

### ISSUE THREAD

**Issue by imajaydwivedi** [imajaydwivedi/SQLMonitor#181]
```
2022Oct20_1957 INFO:      *****Working on step '2__AllDatabaseObjects'..
2022Oct20_1957 INFO:      Creating All Objects in [DBA_Admin] database..
2022Oct20_1957 INFO:      $AllDatabaseObjectsFilePath = 'D:\GitHub-Personal\SQLMonitor\DDLs\SCH-Create-All-Objects.sql'

Column1                                                        
-------                                                        
Error - @current_boundary_value is NULL. So set to 2 Days back.
2022Oct20_1957 INFO:      $UspCollectWaitStatsFilePath = 'D:\GitHub-Personal\SQLMonitor\DDLs\SCH-usp_collect_wait_stats.sql'
2022Oct20_1957 INFO:      $UspCollectXeventsResourceConsumptionFilePath = 'D:\GitHub-Personal\SQLMonitor\DDLs\SCH-usp_collect_xevents_resource_consumption.sql'
Cannot resolve the collation conflict between "SQL_Latin1_General_CP1_CI_AS" and "Latin1_General_CI_AI" in the equal to operation.
At C:\Program Files\WindowsPowerShell\Modules\dbatools\1.1.84\allcommands.ps1:94074 char:9
+         throw $records[0]
+         ~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (SqlPractice:String) [], Exception
    + FullyQualifiedErrorId : dbatools_Invoke-DbaQuery
```

**Comment by paulopires16** [comment]
Ajay, bug solved in this version: v2.0.0 ???
Thanks
Paulo

**Comment by imajaydwivedi** [comment]
Yes. Latest code is having the fix.
